src/Calculator/SettingsPage.xaml.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "SettingsPage.xaml.h"
+#include "CalcViewModel/Common/AppResourceProvider.h"
+#include "CalcViewModel/Common/LocalizationService.h"
+#include "CalcViewModel/Common/LocalizationStringUtil.h"
+#include "CalcViewModel/Common/TraceLogger.h"
+
+using namespace std;
+using namespace CalculatorApp;
+using namespace CalculatorApp::Common;
+using namespace Platform;
+using namespace Windows::ApplicationModel;
+using namespace Windows::Foundation;
+using namespace Windows::System;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Primitives;
+using namespace Windows::UI::Xaml::Data;
+using namespace winrt::Windows::Storage;
+
+#ifndef BUILD_YEAR
+#define BUILD_YEAR 2021
+#endif
+
+SettingsPage::SettingsPage()
+{
+    auto resourceLoader = AppResourceProvider::GetInstance();
+    auto themevalue = Windows::Storage::ApplicationData::Current->LocalSettings->Values->Lookup(L"themeSetting");
+    auto restartValue = Windows::Storage::ApplicationData::Current->LocalSettings->Values->Lookup(L"restartApp");
+    auto currentThemeValue = Windows::Storage::ApplicationData::Current->LocalSettings->Values->Lookup(L"CurrentTheme");
+    String ^ colorSetting = safe_cast<String ^>(themevalue);
+    String ^ restartApp = safe_cast<String ^>(restartValue);
+    String ^ currentTheme = safe_cast<String ^>(currentThemeValue);
+
+    InitializeComponent();
+
+    this->SetVersionString();
+    this->SetCopyrightString();
+    this->InitializeContributeTextBlock();
+    SetDefaultFocus(); 
+    
+    if (themevalue != nullptr)
+    {
+        if (colorSetting == L"Light")
```

Comment:
Consider making these strings a const instead

Reply:
@joseartrivera Sorry, but what string are you referring to? colorSetting?

Reply:
Yes, but if you can make them an enum that would be better. (I posted this before thinking of the enum approach)

Reply:
@joseartrivera I think I've resolved the issue you're referring to, but I'm not quite sure. Could you please check? Thanks.

Reply:
@joseartrivera?

Reply:
Yes, that works also!